account_invoice_validation_queued/views/queue_job_views.xml

```diff
@@ -16,7 +16,7 @@
                         class="oe_highlight"
                         attrs="{'invisible': [('state', '!=', 'failed')]}"/>
                  <button type="object"
-                         name="cancel"
+                         name="unlink"
```

Comment:
You can apply this if you want the same functionality as before, with the `cancel()` method.

```suggestion
                         name="unlink"
                         attrs="{'invisible': [('state', 'not in', ('pending', 'enqueued'))]}"
```

Reply:
It's already put 3 lines below